data/data-pipeline/data_pipeline/etl/score/tests/test_score_post.py

```diff
@@ -89,21 +88,9 @@ def test_create_tile_data(etl, score_data_expected, tile_data_expected):
     )
 
 
-def test_create_downloadable_data(
```

Comment:
I don't understand why we're deleting these tests -- it would seem important to actually run them to make sure the files haven't updated?

Reply:
Addressed these constant changes.

The deletion of the downloadable test is because the moment we open the YAMLs to be editable by Content/Product team, that flexibility breaks this test, since it's expecting a pickle of the CSV output, which is now in control of the YAML file and we can't expect the Content/Product team to generate a Python pickle -- that would break the whole point of this implementation.

Reply:
Hmm. To be honest that makes me pretty nervous to no longer test these outputs. I'm not so much worried about making changes to the code intentionally resulting in errors, I'm more worried about code changes unintentionally changing the output files. 

We could have a couple paths forward. 

1. We could have a practice of having the content team make these edits and hand off to an engineer to update pickles. We probably don't want content people to merge PRs directly anyways. We may also want to prioritize implementing https://github.com/usds/justice40-tool/issues/1048 so that it's easier for engineers to update pickles. 

2. Implement test fixtures for csv.yml and excel.yml so that we generate predictable outputs. 

I think I like option 1 a lot better just for completeness, but could be talked into other paths forward.

Reply:
Let's discuss on our 1x1 tomorrow, but I wanted to be super clear -- I have made no assumption or are implying content/product would merge a branch without an engineer. Content can edit the YAML and preview their download packet, BUT an engineer would have to review the PR code changes and merge to `main` always.

I am torn on this one, though, because the existing test would entail an extra step for the engineer that will slow down the quick update workflow this PR enables. Specially since the content team WILL check the output of the code in the downloadable packet anyway, why do we need to test it?

But I'm open to be swayed.